food-truck-api/src/main/java/food/truck/api/truck/Truck.java

```diff
@@ -0,0 +1,41 @@
+package food.truck.api.truck;
+
+import lombok.Data;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+@Table(name = "truck")
+public class Truck {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "truck_id", nullable = false)
+    Long id;
+
+    @JoinColumn(foreignKey = @ForeignKey(name = "user_id"), nullable = false)
+    Long userId;
+
+    @Column(name = "name", nullable = false)
+    String name;
+
+    // TODO: store actual menu, potentially an image?
```

Comment:
I don't think using a Java Image will be useful here, since it'd need to go back to bytes to send it client side.

Reply:
I wasn't necessarily suggesting an image object, but I was thinking we might be able to store image URLs. If we wanted to really go above and beyond we could try to set up an Amazon S3 bucket for that (I think that's a common use case for it?), though I have no experience with that.